src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void addList(String line) {
+        System.out.println("--------------------------------------");
```

Comment:
I feel like using a constant name variable is better in this case so that the entire presentation feels cleaner without all the dashes. Such as using static String BREAK_LINE = "________________________________________________".